### ISSUE TITLE
Use h1 tags instead of h2 for admin page titles

### ISSUE THREAD

**Issue by stephenharris** [stephenharris/Event-Organiser#282]
In WordPress 4.3+ page titles use h1 tags rather than h2. Admin pages should have their title tags updated accordingly, and the possibility of maintaining backwards compatibility with older versions of WordPress considered. This would be a good opportunity for a wider review of page structures
